bash_exam.sh

```diff
@@ -1,49 +1,70 @@
 # Set/Select this file as a bash/shell script - you must set a header of the file
-# TODO
+#1 /bin/bash
 
 # Add announcement that this script has just started with timestamp
-# TODO
+echo "Script has started at $(date +%T)"
 
 # Create a new "working_dir" variable and set it as a current directory
-#TODO
+working_dir=$(pwd)
+echo "Current working directory: $working_dir"
 
 # In your working dir, create nested directory structure "src/main/module" (discover the -p argument in mkdir command)
-# TODO
+mkdir -p src/main/module
 
 # Create a new empty file ".gitkeep" in the "module" folder created in previous step
-# TODO
+cd src/main/module
+touch .gitkeep
 
 # List the "module" directory mentioned in the previous step and the result store to a new variable
-# TODO
+module_dir_list=$(ls -a)
 
 # Check if the "module" directory is empty. If so or not, show a meaningful message about it
-# TODO
+if [[ $module_dir_list == "" ]] 
+then
+	echo "Module directory is empty"
+else 
+	echo "Module directory is not empty"
+fi 
 
 # In your working directory create a new empty file "setup.ini"
-# TODO
+cd "$working_dir"
+touch setup.ini
 
 # You noticed you had wanted to create the "setup.ini" file to "module" folder, but you placed it to the working directory.
 # So move the file there
-# TODO
+mv setup.ini src/main/module
 
 # Your teammate warned you that the "src/main" is not standardized path and he asked you to remove it.
 # The content must be saved, and placed in the working directory
-# TODO
+cp -R src/main/module "$working_dir"
+rm -rf src
 
 # Fill "setup.ini" file with "user=admin" value
-# TODO
+cd module
+echo "user=admin" > setup.ini
+setup_ini_content=$(cat setup.ini)
 
 # Print the content of the file from previous step for check and add a message about the file name before (A user always wants to be informed what's happening)
-# TODO
+echo "Following content has been added to setup.ini file: $setup_ini_content"
```

Comment:
It works for one-line files well. But how readable it would be in case of many-line files?

The file content should be placed on new line.